### PR TITLE
Improve Lighthouse performance score

### PR DESCRIPTION
**Pull request by ravise5**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by ravise5** [comment]
Lighthouse documentation recommends computing median of 5 runs to reduce variability in the scores
refer: https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md#run-lighthouse-multiple-times

File: it/config/src/main/content/jcr_root/apps/system/config/org.apache.felix.http.cfg.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "org.apache.felix.jetty.gziphandler.enable":true,
+  "org.apache.felix.jetty.gzip.includedMimeTypes":[
```
Comment (rismehta) [inline it/config/src/main/content/jcr_root/apps/system/config/org.apache.felix.http.cfg.json:3]: You need to only whitelist a particular URL and not URL, please fix this.

**Review by rismehta — CHANGES_REQUESTED** [review]
-1 for now

File: it/config/src/main/content/jcr_root/apps/system/config/org.apache.felix.http.cfg.json (hunk 1 above)
Comment (ravise5) [reply it/config/src/main/content/jcr_root/apps/system/config/org.apache.felix.http.cfg.json]: whitelisted the collateral path in config

**Comment by rismehta** [comment]
Merging this, since test failures are being worked upon by @sakshi-arora1

**Review by rismehta — APPROVED** [review]

**Pull request merged**